src/main/java/org/gridsuite/explore/server/ExploreController.java

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.explore.server;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.codec.multipart.FilePart;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com>
+ */
+@RestController
+@RequestMapping(value = "/" + ExploreApi.API_VERSION)
+@Tag(name = "explore-server")
+public class ExploreController {
+
+    private final ExploreService exploreService;
+
+    public ExploreController(ExploreService exploreService) {
+        this.exploreService = exploreService;
+    }
+
+    @PostMapping(value = "/directories/studies/{studyName}/cases/{caseUuid}")
```

Comment:
Not sure if it should be `/explore/...` in all URLs here instead of `/directories/...`
Because it has been moved from directory server but does the path here is still consistent ?

Reply:
Done